deploy.sh

```diff
@@ -78,10 +63,10 @@ function preprocess_and_publish {
   # commit to destination branch with shorthash in message
   git commit -am "update using ${SOURCE_BRANCH}/${shorthash}" || abort "Failed to commit to destination branch '${DEPLOY_BRANCH}'"
 
-  # the "-u" here is if the branch was created
-  git push -u "${GH_REMOTE}" "${DEPLOY_BRANCH}" || abort "Failed to push to '${GH_REMOTE}/${DEPLOY_BRANCH}'"
+  # executes a forced "git subtree push" (git subtree does not take a force so we have to nest)
+  git push "${GH_REMOTE}" `git subtree split --prefix "${WEBSITE_DIR_PATH}" "${SOURCE_BRANCH}"`:"${DEPLOY_BRANCH}" --force || abort "Failed to push to '${GH_REMOTE}/${DEPLOY_BRANCH}'"
```

Comment:
Shellcheck has a few complaints about this line (the use of backticks [SC2006](https://github.com/koalaman/shellcheck/wiki/SC2006) and the lack of quotes [SC2046](https://github.com/koalaman/shellcheck/wiki/SC2046))